pages/team.html

```diff
@@ -0,0 +1,90 @@
+---
+title: Impact Labs - Team
+permalink: /team
+---
+
+<!-- Temp styling in file, will be added in SCSS files -->
+<style>
```

Comment:
Future note: Could probably chuck this in the scss file now 😆

Reply:
yup sure, might have to clean up the scss eventually